DP1.10081.002/Standard.calculations.txt

```diff
@@ -0,0 +1 @@
+An R package, phyloNEON, has been developed to convert the downloaded MCT taxonomy tables to a format ready to import into Phyloseq (https://joey711.github.io/phyloseq/), one of the most popular packages to analyze amplicon sequence data. Basic instructions to install the package and convert the downloaded data are available on the GitHub repository page (https://github.com/NEONScience/phyloNEON). More detailed instructions and tutorials will be released in the coming months, including converting the NEON MCT data to other popular formats (e.g. Qiime2), and add related NEON data to the sample metadata table for analysis. 
```

Comment:
Are there other ways people might analyze these data? To me this reads as though every user will want to use Phyloseq, and I'm not sure that's true. Is there anything more general to say here? I'm thinking something about how to link to the soil data to get environmental conditions the samples were taken from.

Leave out the final sentence about what is upcoming, comments like that have a tendency to cause us trouble down the line. We can add more links here in the future as needed.

Reply:
I have changed the text to not just reference Phyloseq, and also focus on the BIOM format, which is more universal. The text should now not assume that everyone will use Phyloseq. I have also added a paragraph to help users build on associated NEON metadata. This was tricky, as the MCT sample metadata table does not include the parentID. We should consider adding this field. For the near term, I will add instructions to the README to derive the parentID from the dnaSampleID, so as to open more ways to add metadata.